app-games/src/main/java/cm/aptoide/pt/app_games/home/BundlesView.kt

```diff
@@ -25,7 +166,7 @@ fun BundlesView(
       .fillMaxSize()
       .wrapContentSize(Alignment.TopCenter)
   ) {
-    if (viewState.type == LOADING) {
+    if (viewState.type == BundlesViewUiStateType.LOADING) {
```

Comment:
Remove this condition. The loading state is already handled in the parent composable